### PR TITLE
Rtcp octo

### PR DESCRIPTION
**Pull request by bgrozev**

### CODE REVIEW

File: src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java [hunk 1]
```diff
@@ -149,7 +147,7 @@ private static void resolveSSRCAsEndpoint(
      * The <tt>DominantSpeakerIdentification</tt> instance which
      * detects/identifies the active/dominant speaker in {@link #conference}.
      */
-    private final DominantSpeakerIdentification dominantSpeakerIdentification
+    private DominantSpeakerIdentification dominantSpeakerIdentification
```
Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java]: not opposing, just curious why?

File: src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java [hunk 2]
```diff
@@ -198,9 +184,6 @@ public ConferenceSpeechActivity(Conference conference)
         this.conference = Objects.requireNonNull(conference, "conference");
         logger = Logger.getLogger(classLogger, conference.getLogger());
 
-         // The PropertyChangeListener will weakly reference this instance and
-         // will unregister itself from the conference sooner or later.
-        conference.addPropertyChangeListener(propertyChangeListener);
         dominantSpeakerIdentification
```
Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java]: i think it's safe to leave as, but i'm surprised we check for null elsewhere

File: src/main/java/org/jitsi/videobridge/cc/BitrateController.java [hunk 3]
```diff
@@ -829,12 +862,13 @@ private synchronized void update()
...
             {
-                logger.debug("TEMP: adding a new track projection for ssrc" +
-                        rtpEncoding.getPrimarySSRC());
                 adaptiveTrackProjectionMap.put(
                     rtpEncoding.getPrimarySSRC(),
                     adaptiveTrackProjection);
 
+                // TODO: RTX packets should never reach here (they get de-RTXed
+                // in the receiver chain). Do we still need this mapping?
+                // Perhaps we need it in order to block SR for the RTX streams?
```
Comment (gpolitis) [inline src/main/java/org/jitsi/videobridge/cc/BitrateController.java]: that's a really good point, in theory it should be safe to remove

Comment (bgrozev) [reply src/main/java/org/jitsi/videobridge/cc/BitrateController.java]: Do you know whether RTX streams have SRs?

File: src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java (hunk 1 above)
Comment (bgrozev) [reply src/main/java/org/jitsi/videobridge/ConferenceSpeechActivity.java]: I must have missed something while rebasing. I want to set it to null in `expire`.

File: src/main/java/org/jitsi/videobridge/cc/BitrateController.java (hunk 3 above)
Comment (gpolitis) [reply src/main/java/org/jitsi/videobridge/cc/BitrateController.java]: not in practice at least

**Pull request merged**